Cargo.toml

```diff
@@ -58,6 +58,7 @@ nostr = { version = "0.18.0", default-features = false, features = ["base", "nip
 [target.'cfg(not(target_env = "msvc"))'.dependencies]
 tikv-jemallocator = "0.5"
 log = "0.4"
+cln-rpc = "0.1.9"
```

Comment:
utACK 45dbb8ea1f7c19f0ebe0c23cd1de7e0a41fd8e7d

Only comment is it might be nice id the exta dep could be optional behind a feature